### DISCUSSION TITLE
Xray Oxygen alive and its features have been moved in OpenXRay?

### DISCUSSION THREAD

**Discussion by GermanAizek** [OpenXRay/xray-16#1138] [Q&A] [answered]
@Xottab-DUTY 
Where has active XRay Oxygen fork? What happened to developers including ForserX?

I find old fork here: https://github.com/xrLil-Batya/xray-oxygen

Previously there was ForserX activity on gameru.net: https://www.gameru.net/forum/index.php?showuser=22151

**Comment by Xottab-DUTY** [comment, marked as answer]
Project moved to closed-source development.

As for the features, there are many of them in Oxygen, but we didn't work on merging them. We are open for pull requests :)

**Reply by aka-sektor** [reply]
Чёт пропустил момент когда ПЫС прихлопнул Оксигена

**Reply by Xottab-DUTY** [reply]
> Чёт пропустил момент когда ПЫС прихлопнул Оксигена

Никто их не хлопал, они ушли в закрытую разработку сами, проект разрабатывается до сих пор.

**Reply by aka-sektor** [reply]
А никаких собранных версий движка или исходников не осталось? 20 июня 2019 года ещё все было

**Reply by Hrusteckiy** [reply]
Есть форки, из них и собери.

**Reply by aka-sektor** [reply]
> Есть форки, из них и собери.

Так подскажи ссылки, где чего.

**Reply by Xottab-DUTY** [reply]
> > Есть форки, из них и собери.
> 
> Так подскажи ссылки, где чего.

Так в самом начале этой темы ссылка буквально.
Там правда написано old fork, так что я поискал на гитхабе. Нашёл только ещё более старые, так что будем считать, что этот новый :)
Окси как раз в 2020-м, где-то, и перешли в закрытую разработку.
